glue_jupyter/bqplot/common/tools.py

```diff
@@ -25,6 +25,24 @@ def deactivate(self):
         self.viewer._mouse_interact.next = None
 
 
+class BqplotSelectionTool(InteractCheckableTool):
+    """Disables all other tools when activated."""
+
+    def activate(self):
+        with self.viewer._output_widget:
+            self.interact.selected_x = None
+            self.interact.selected_y = None
+
+        for viewer in self.viewer.session.application.viewers:
+            if viewer is not self.viewer:
+                viewer.toolbar.active_tool = None
+
+        # TODO: Tom asked me to add this, but adding this fails the test.
+        self.viewer.session.edit_subset_mode.edit_subset = []
```

Comment:
@astrofrog , I uncommented this line as you requested. CI will fail but I have no idea how to fix it. Please advise. Thanks!